### DISCUSSION TITLE
How to get HiTrack files using Emu like BlueStacks+ or MEMu

### DISCUSSION THREAD

**Discussion by thechurem** [CTHRU/Huawei-TCX-Converter#60] [Show and tell]
I have installed it and root them
but HiTrack files a no longer kept in /data/data/com_huawei_health/files folder.
my health ver is 10.0.2.311

**Comment by thechurem** [comment]
requesting data from Huawei very longtime

**Comment by HardCorePawn** [comment]
They seem to be showing up just fine when using Bluestacks (and Health v. 10.1.1.312) for me:
![file listing from BSTweaker](https://i.imgur.com/Az0Qprc.png)

NOTE: I'm accessing the file system using the "File Manager" in BlueStacks Tweaker

**Comment by thechurem** [comment]
thx for your kind sir. Problem solve. You made my day.
I see in Huawei forum they use time to develop export feature for many One or Two year. He is not sincere to customers.
Ctru and Hitrava can help us. 
next year I will sell my Huawei gadget for my social and tracking . 
 

> They seem to be showing up just fine when using Bluestacks (and Health v. 10.1.1.312) for me:
> ![file listing from BSTweaker](https://camo.githubusercontent.com/3ba59e8523e711c9df6e6db015cabc66e5c1ee5a853e155013795ac6727eb2f9/68747470733a2f2f692e696d6775722e636f6d2f417a30517072632e706e67)
> 
> NOTE: I'm accessing the file system using the "File Manager" in BlueStacks Tweaker
